### ISSUE TITLE
Generates an invalid code block when code has an empty line

### ISSUE THREAD

**Issue by vzvu3k6k** [azu/gitbook-plugin-include-codeblock#3]
Let fixture/test.js has the following content:

``` js
console.log("foo");

console.log("bar");
```

This plugin will expand `[import](fixtures/test.js)` into:

``````
> [test.js](fixtures/test.js)
```js
console.log("foo");

console.log("bar");
```
``````

Gitbook renders it as:

`````` html
<blockquote>
<p><a href="fixtures/test.js">test.js</a>
```js
console.log(&quot;foo&quot;);</p>
</blockquote>
<p>console.log(&quot;bar&quot;);</p>
<pre><code>
``````

These pages are affected by this issue:
- http://azu.gitbooks.io/javascript-plugin-architecture/content/ja/jQuery/index.html
- http://azu.gitbooks.io/javascript-plugin-architecture/content/ja/ESLint/index.html

**Comment by azu** [comment]
@vzvu3k6k Thanks for report. 
This isssue was fixed in #4

**Comment by vzvu3k6k** [comment]
:+1:
